### PR TITLE
[hi] Localized docs/concepts/containers/container-lifecycle-hooks.md

### PR DESCRIPTION
**Pull request by aj11anuj**

### CODE REVIEW

File: content/hi/docs/concepts/containers/container-lifecycle-hooks.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
+---
+reviewers:
+- Babapool
+- divya-mohan0209
```
Comment (Babapool) [inline content/hi/docs/concepts/containers/container-lifecycle-hooks.md:4]: I don't think this reviewer block should apply to the localization, so we can remove it.
```suggestion
```

File: content/hi/docs/concepts/containers/container-lifecycle-hooks.md [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+title: कंटेनर जीवनचक्र हुक
+content_type: concept
+weight: 40
+---
+
+<!-- overview -->
+
+यह पृष्ठ वर्णन करता है कि कैसे kubelet कंटेनरों को संभालता है। 
+उनके प्रबंधन जीवनचक्र के दौरान घटनाओं द्वारा ट्रिगर किए गए कोड को चलाने के लिए 
+जीवनचक्र हुक फ़्रेमवर्क का उपयोग करता है।
```
Comment (Babapool) [inline content/hi/docs/concepts/containers/container-lifecycle-hooks.md:14]:
```suggestion
यह पृष्ठ वर्णन करता है कि क्यूबलेट द्वारा प्रबंधित कंटेनर अपने प्रबंधन जीवनचक्र में घटनाओं द्वारा ट्रिगर किए गए निर्दिष्ट कोड को चलाने के लिए कंटेनर जीवनचक्र कॉलबैक ढांचे का उपयोग कैसे करते हैं।
```

File: content/hi/docs/concepts/containers/container-lifecycle-hooks.md [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+
+
+<!-- body -->
+
+## अवलोकन
+
+जेसे कई प्रोग्रामिंग लैंग्वेज के फ्रेमवर्क के पास एंगुलर जेसे कंपोनेंट्स जीवनचक्र के हुक होते हैं, 
+वेसे ही कुबेरनेट्स कंटेनरों को जीवनचक्र हुक प्रदान करता है।
```
Comment (Babapool) [inline content/hi/docs/concepts/containers/container-lifecycle-hooks.md:22]:
```suggestion
कई प्रोग्रामिंग भाषा फ्रेमवर्क के अनुरूप, जिनमें घटक जीवनचक्र हुक होते हैं, जैसे कि Angular, कुबेरनेट्स कंटेनरों को जीवनचक्र हुक प्रदान करता है। 
```

File: content/hi/docs/concepts/containers/container-lifecycle-hooks.md [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+
+दो हुक जो कंटेनरों के संपर्क में होते हैं:
+
+'PostStart'
+कंटेनर बनने के तुरंत बाद इस हुक को चालू किया जाता है। 
+हालांकि, इस बात की कोई गारंटी नहीं है कि हुक कंटेनर ENTRYPOINT से पहले चालू होगा। 
+हैंडलर को कोई पैरामीटर नहीं दिया जाता।
+'PreStop'
+
+एपीआई रिक्वेस्ट या प्रबंधन के स्टार्टअप जांच मे विफलता, पूर्वावलोकन, संसाधन विवाद और
```
Comment (Babapool) [inline content/hi/docs/concepts/containers/container-lifecycle-hooks.md:37]:
```suggestion
API अनुरोध या प्रबंधन के लाइवनेस/स्टार्टअप जांच मे विफलता, पूर्वावलोकन, संसाधन विवाद और 
```

File: content/hi/docs/concepts/containers/container-lifecycle-hooks.md [hunk 5]
```diff
@@ -0,0 +1,133 @@
...
+दो हुक जो कंटेनरों के संपर्क में होते हैं:
+
+'PostStart'
+कंटेनर बनने के तुरंत बाद इस हुक को चालू किया जाता है। 
+हालांकि, इस बात की कोई गारंटी नहीं है कि हुक कंटेनर ENTRYPOINT से पहले चालू होगा। 
+हैंडलर को कोई पैरामीटर नहीं दिया जाता।
+'PreStop'
+
+एपीआई रिक्वेस्ट या प्रबंधन के स्टार्टअप जांच मे विफलता, पूर्वावलोकन, संसाधन विवाद और 
+अन्य जैसी घटनाएं के कारण कंटेनर समाप्त होने से तुरंत पहले इस हुक को रिक्वेस्ट भेजी जाती है।
```
Comment (Babapool) [inline content/hi/docs/concepts/containers/container-lifecycle-hooks.md:38]:
```suggestion
अन्य जैसी घटनाएं के कारण कंटेनर समाप्त होने से तुरंत पहले इस हुक को अनुरोध भेजी जाती है।
```

**Review by Babapool — COMMENTED** [review]
Some suggestions from myside, hope you find them useful. Thanks

**Comment by utkarsh-singh1** [comment]
Hi @aj11anuj, Can you address the review in the PR?

**Comment by kundan2707** [comment]
@aj11anuj  can you address review comments given?

**Comment by kundan2707** [comment]
@aj11anuj gentle reminder

**Comment by kundan2707** [comment]
closing due to long time inactivity.

**Comment by kundan2707** [comment]
@aj11anuj  feel free to reopen if you have time to work on this.

**Pull request closed without merging**